src/DynamoCoreWpf/ViewModels/PackageManager/PackageManagerClientViewModel.cs

```diff
@@ -64,7 +64,7 @@ public TermsOfUseHelper(TermsOfUseHelperParams touParams)
         }
 
 
-        public void Execute(bool preventReentrant = true)
+        internal void Execute(bool preventReentrant = true)
```

Comment:
I initially thought that `TermsOfUseHelper` should never be used by external developer in any way, especially since it shows the terms of use dialog of Autodesk. But then again _if_ for some unimaginable reasons some external components make use of it, it will become a problem for our release and we'll have people pointing fingers at us. So I think it is still best to keep the original `Execute` method around for compatibility sake, and do this:

```csharp
[Obsolete("Please use the other overridden method")]
public void Execute() {
    Execute(true); // Redirected to call the alternative method.
}

internal void Execute(bool preventReentrant) { // No default value.
    // The original implementation goes here...
}
```